storio-sqlite/src/test/java/com/pushtorefresh/storio3/sqlite/integration/InsertTest.java

```diff
@@ -22,7 +24,7 @@ public void insertOne() {
         final User user = putUserBlocking();
 
         // why we created StorIOSQLite: nobody loves nulls
```

Comment:
This comment is now outdated :sweat_smile:

Reply:
hah 👍